softwareComponents/simplesimServerLib/include/simplesim_server.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <cassert>
+#include <filesystem>
+#include <ostream>
+
+#include <dimcli/cli.h>
+
+// RoFI headers
+#include <configuration/rofibot.hpp>
+#include <configuration/serialization.hpp>
+#include <configuration/universalModule.hpp>
+#include <simplesim/packet_filters/py_filter.hpp>
+
+
+namespace rofi::simplesim
+{
+enum class ConfigurationFormat
+{
+    Old = -1,
+    Json,
+};
+
+inline auto operator<<( std::ostream & ostr, ConfigurationFormat configFormat ) -> std::ostream &
+{
+    switch ( configFormat ) {
+        case ConfigurationFormat::Old:
+            return ostr << "old";
+        case ConfigurationFormat::Json:
+            return ostr << "json";
+    }
+    assert( false );
+    return ostr << static_cast< int >( configFormat );
```

Comment:
Místo `assert( false )` bychom si měli vyrobit nějakou trap funkci, která:
- v release volá `__bultin_unreachable`/`__builtin_trap`
- v debug módu triggeruje assert.

Reply:
to ale není kód, co by kdy běžel na Rofíku (nemá gazebo...)

Reply:
No jasně, ale jak se to chová v release módu?

Reply:
Přidaná `ROFI_UNREACHABLE` v `atoms/unreachable.hpp`